core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -375,12 +375,9 @@ public void configureBaseApplication(Configurable<?> config) {
    * client.
    */
   public void configureBaseApplication(Configurable<?> config, Map<String, String> metricTags) {
-    RestConfig restConfig = getConfiguration();
+    registerJsonProvider(config, true);
```

Comment:
If we're going to provide methods for some of these bits, wouldn't it make sense to break out all of the substeps? e.g. `registerJsonProvider`, `registerFeatures`, `registerExceptionMappers`, etc?